src/User.js

```diff
@@ -10,7 +10,9 @@ class User {
   }
 
   showFirstName() {
-    const firstName = this.name.split(' ').slice(0)
+    return this.name.split(' ')[0]
   }
 
-}
\ No newline at end of file
+}
+
+module.exports = User
```

Comment:
I think this might need a semicolon at the end of it... given that red dot at the end of the line. First time seeing that.